packages/autocomplete-preset-algolia/src/requester/getAlgoliaFacets.ts

```diff
@@ -7,6 +9,11 @@ import { RequestParams } from './createRequester';
  * Retrieves Algolia facet hits from multiple indices.
  */
 export function getAlgoliaFacets<TTHit>(requestParams: RequestParams<TTHit>) {
+  invariant(
+    typeof requestParams.searchClient === 'object',
+    'The `searchClient` option is required for getAlgoliaFacets({ searchClient }).'
```

Comment:
It's more a param than an option, no?

```suggestion
    'The `searchClient` parameter is required for getAlgoliaFacets({ searchClient }).'
```